packages/plugin-react-refresh/index.d.ts

```diff
@@ -6,7 +6,8 @@ type PluginFactory = (options?: Options) => Plugin
 declare const createPlugin: PluginFactory & { preambleCode: string }
 
 export interface Options {
-  parserPlugins: ParserOptions['plugins']
+  parserPlugins?: ParserOptions['plugins']
+  exclude?: (filename: string) => boolean
```

Comment:
The type here need to be updated

Reply:
Oh right, I forgot the types are separate in this repo, thanks.